<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="PortBox"/>
      </item>
      <item row="2" column="7" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>源地址</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="QLineEdit" name="destinationAddrlineEdit"/>
      </item>
      <item row="1" column="15" rowspan="6" colspan="2">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>目标地址</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>设备地址</string>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>命令</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="7" colspan="6">
       <widget class="QLineEdit" name="SourceAddrlineEdit"/>
      </item>
      <item row="3" column="13" colspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QPushButton" name="ReadParaButton">
        <property name="text">
         <string>读参数</string>
        </property>
       </widget>
      </item>
      <item row="5" column="8" colspan="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="4" colspan="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QLineEdit" name="DevAddrlineEdit"/>
      </item>
      <item row="5" column="6" colspan="2">
       <widget class="QComboBox" name="CmdcomboBox"/>
      </item>
      <item row="5" column="14">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="10" colspan="4">
       <widget class="QPushButton" name="SendpushButton">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
      <item row="6" column="12" colspan="3">
       <widget class="QPushButton" name="ReadDatapushButton">
        <property name="text">
         <string>读数据</string>
        </property>
       </widget>
      </item>
      <item row="6" column="7" colspan="3">
       <widget class="QPushButton" name="CheckpushButton">
        <property name="text">
         <string>校表</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3" colspan="4">
       <widget class="QPushButton" name="writeParaButton">
        <property name="text">
         <string>写参数</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" colspan="2">
       <widget class="QComboBox" name="BaudBox">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>串口号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
      </item>
      <item row="0" column="16">
       <widget class="QPushButton" name="ClearpushButton">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item row="0" column="15">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9" rowspan="2" colspan="2">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="11">
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
